src/App.tsx

```diff
@@ -13,28 +13,28 @@ import RequireData from './components/RequireData';
 const App: React.FC = () => {
   return (
     <HashRouter>
-      <Template>
-        <Switch>
-          <Route path="/" exact render={() => (
-            <RequireData rpc activeWallet component={Wallet} />
-          )} />
-          <Route path="/send" render={() => (
-            <RequireData rpc activeWallet component={Send} />
-          )} />
-          <Route path="/sign" render={() => (
-            <RequireData rpc activeWallet component={Sign} />
-          )} />
-          <Route path="/history" render={() => (
-            <RequireData rpc activeWallet component={History} />
-          )} />
-          <Route path="/create" render={() => (
-            <RequireData rpc component={Create} />
-          )} />
-          <Route path="/settings" component={Settings} />
-        </Switch>
-      </Template>
+      <Switch>
```

Comment:
I switched these while attempting to figure out bootstrap's component hierarchy, but left the change because it makes more sense. Perhaps I add routes in the future that belong in the `<Switch/>` router component but don't have the `<Template/>` formatting ...